src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -155,28 +155,28 @@ them, and how to use them to configure your environment.
     some reason to handle multiple EC2 accounts. Other properties in the
     *Create Node Source* are:
     
-    -   *rmHostname:* The hostname or the public IP address of the resource manager. This address needs to be accessible from the AWS cloud. 
+    -   *rmHostname:* The hostname or the public IP address of the resource manager. This address needs to be accessible from the AWS cloud.
     
-    -   *connectorIaasURL:* Connector-iaas is a service embedded in the scheduler used to interact with IaaS like AWS. By default it runs on the following url rmHostname/connector-iaas.    
+    -   *connectorIaasURL:* Connector-iaas is a service embedded in the scheduler used to interact with IaaS like AWS. By default it runs on the following url rmHostname/connector-iaas.
 
-    -   *image:* Defines which AMI will be deployed on the AWS instances to create. The
-        value to provide is the AWS region together with the unique AMI Id, for example: `eu-west-1/ami-bff32ccc`. 
+    -   *image:* Defines which AMI will be used to create an AWS instances. The
+        value to provide is the AWS region together with the unique AMI Id, for example: `eu-west-1/ami-bff32ccc`.
         
-    -   *numberOfInstances:* Total number of AWS instances to create for this infrastructure. 
+    -   *numberOfInstances:* Total number of AWS instances to create for this infrastructure.
     
-    -   *numberOfNodesPerInstance:* Total number of Proactive Nodes to deploy in each AWS created instance. 
+    -   *numberOfNodesPerInstance:* Total number of Proactive Nodes to deploy in each AWS created instance.
     
 +
 TIP: If all the nodes of an AWS instance are removed, the instance will be terminated. For more information on the terminated state in AWS please see http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/terminating-instances.html[AWS Terminating Instances].
 +
        
-    -   *downloadCommand:* The command to download the Proactive node.jar. This command is executed in all the newly created AWS instances at cloud init time. 
-    	The full URL path of the node.jar to download, needs to be accessible from the AWS cloud.     	
+    -   *downloadCommand:* The command to download the Proactive node.jar. This command is executed in all the newly created AWS instances at cloud init time.
```

Comment:
Capitalize _cloud init_ and make reference to CloudInit documentation?

https://cloudinit.readthedocs.org/en/latest/

It could be helpful to get more details.